boards/arm/olimex_stm32_h103/olimex_stm32_h103_defconfig

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32F1X=y
+CONFIG_SOC_STM32F103XB=y
+# 72MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=72000000
+
+# enable uart driver
+CONFIG_SERIAL=y
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable pinmux
+CONFIG_PINMUX=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+# Clock configuration for Cube Clock control driver
```

Comment:
This comment should be clarified, as there is no `Cube Clock control driver`. Maybe meant `Clock configuration using CubeMX suggested defaults`?

Reply:
Will remove the comment, it has nothing to do with this board